pyaedt/generic/LoadAEDTFile.py

```diff
@@ -161,17 +161,15 @@ def _decode_value_and_save(k, v, d):
             d[k] = _parse_value(v)
 
 
-def _decode_key(l, d):
+def _decode_key(l: str, d) -> None:
```

Comment:
Type checking isn't permitted since we need to import on IronPython 2.7

Reply:
Yes, please revert it. 

```suggestion
def _decode_key(l, d):
```